src/style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 0;
+}
+
+.product_card {
+  font-family: Roboto, Arial, sans-serif;
+  width: 199px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card_img {
+  width: 160px;
+  height: 134px;
+  margin-left: 18px;
+  margin-top: 32px;
+  margin-bottom: 20px;
+}
+
+.card_item {
+  margin-top: 16px;
+  width: 166px;
+  height: 36px;
+  margin-left: 16px;
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.card_id {
+  display: flex;
+  justify-content: center;
+  width: 94px;
+  height: 14px;
+  margin-left: 16px;
+  margin-bottom: 16px;
+  top: 90px;
+  font-weight: 400;
+  font-size: 10px;
+  font-style: normal;
+
+  color: rgba(97, 96, 112, 1);
+}
+
+.card_review {
+  display: flex;
+  margin-top: 0;
+  margin-right: 16px;
+  margin-left: 16px;
+  margin-bottom: 24px;
+  height: 16px;
+}
+
+.card_review_stars {
+  display: flex;
+
+}
+
+.stars_star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  margin-left: 3px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card_review_stars > .stars_star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card_review_stars > .stars_star:nth-child(-n+4):hover {
+  cursor: pointer;
+}
+
+.card_review_feedback {
+  display: flex;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  align-items: center;
+  margin-left: 17px;
```

Comment:
better put elements to the center of the parent block.
use `text-align center`  or flex for the parent block